test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,38 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsAreNotNull() throws Exception {
+        // empty list
+        assertHasNoNull();
+
+        // only one object
+        assertHasAnyNull((Object) null);
+        assertHasNoNull(1);
+        assertHasNoNull("");
+        assertHasNoNull("abc");
+
+        // all objects unique
+        assertHasNoNull("abc", "ab", "a");
+        assertHasNoNull(1, 2);
+        assertHasAnyNull("abc", "ab", "a", null);
+        assertHasAnyNull(1, 2, null);
+        assertHasAnyNull(null, "a");
+        assertHasAnyNull(null, null);
+        assertHasAnyNull(null, 1, "a", null);
+    }
+
+
+    public void assertHasAnyNull(Object... objects) {
```

Comment:
If this test util method is only in this class, can convert it to private.

Reply:
i see